docs/TestingGanacheProxyModel.rst

```diff
@@ -94,7 +94,7 @@ To run Ganache with Hyperledger Avalon, follow these steps:
    .. code:: sh
 
        cd $TCF_HOME
-       docker-compose -f docker-compose-eth-ganache.yaml up -d --build
+       docker-compose -f docker-compose.yaml -f docker-compose-eth-ganache.yaml up -d --build
```

Comment:
There are few other files to modify - docker/k8s/README.md

Reply:
docker/k8s/fabric/README.md also needs to be changed

Reply:
Done.